model/error/event.go

```diff
@@ -208,6 +224,10 @@ func (e *Event) fields(tctx *transform.Context) common.MapStr {
 	e.data = common.MapStr{}
 	e.add("id", e.Id)
 
+	if tctx.Metadata.User != nil {
+		e.add("user_agent.original", tctx.Metadata.User.UserAgent)
```

Comment:
the `user_agent` needs to be set somewhere within the `fields`, otherwise the `user_agent` pipeline doesn't work.